hub/@hash/shuffle.json

```diff
@@ -0,0 +1,7 @@
+{
+  "repository": "https://github.com/hashintel/hash.git",
+  "commit": "98818b692742dbadd77eaf824c6d51bb8dec7998",
+
+  "workspace": "@hashintel/block-shuffle",
```

Comment:
@luisbettencourt from the [deployment build error](https://vercel.com/hashintel/blockprotocol/6z9a9RRn4hmCK5L3XGefYr53LGNB), looks like `@hashintel/block-shuffle` needs to be added as a yarn workspace to the [`hashintel/hash` repo](https://github.com/hashintel/hash)

Reply:
Actually looks like you just need to rename the workspace from `@hashintel/shuffle` to `@hashintel/block-shuffle`

Reply:
Yes, my bad. Was just waiting for the CI to confirm this